gax-java/gax-grpc/src/main/java/com/google/longrunning/stub/OperationsStubSettings.java

```diff
@@ -243,13 +242,13 @@ public static class Builder extends StubSettings.Builder<OperationsStubSettings,
       RetrySettings settings = null;
       settings =
           RetrySettings.newBuilder()
-              .setInitialRetryDelay(Duration.ofMillis(100L))
+              .setInitialRetryDelayDuration(java.time.Duration.ofMillis(100L))
               .setRetryDelayMultiplier(1.3)
-              .setMaxRetryDelay(Duration.ofMillis(60000L))
-              .setInitialRpcTimeout(Duration.ofMillis(90000L))
+              .setMaxRetryDelayDuration(java.time.Duration.ofMillis(60000L))
+              .setInitialRpcTimeoutDuration(java.time.Duration.ofMillis(90000L))
               .setRpcTimeoutMultiplier(1.0)
-              .setMaxRpcTimeout(Duration.ofMillis(90000L))
-              .setTotalTimeout(Duration.ofMillis(600000L))
+              .setMaxRpcTimeoutDuration(java.time.Duration.ofMillis(90000L))
+              .setTotalTimeoutDuration(java.time.Duration.ofMillis(600000L))
```

Comment:
Same state with this, but this is for gRPC's Operation Client. The gRPC Operatiosn Client should be regenerated at some point, but I think I'm fine with us making these manual changes for now. @blakeli0